tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -365,6 +365,20 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         }
       }
     </style>
+    <template id="colabNotSupported" preserve-content>
+      <div style="max-width: 540px; margin: 80px auto 0 auto;">
+        <h3>Dynamic plugin isn’t supported in Colab yet.</h3>
+        <p>
+          Please see
+          <a
+            href="https://github.com/tensorflow/tensorboard/issues/1913"
```

Comment:
Do you plan to broaden this issue from the profile plugin to “plugins
that use iframes” or similar? I’d be confused if I happened upon this
issue from the projector plugin.